Backpack/src/main/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarController.kt

```diff
@@ -64,6 +64,13 @@ abstract class BpkCalendarController {
     return CalendarDay(year, month, day).date == CalendarDay().date
   }
 
+  fun updateSelection(range: CalendarRange) {
```

Comment:
Is there any use-cases currently that require to programatically set the range/selection? I am just a bit doubtful if it is clear what the purpose of this is when now there is two implementation that's updating selection:
- `onDayOfMonthSelected(selectedDay: CalendarDay)`
- `updateSelection(range: CalendarRange)`

And one that the implementation needs to define: `onRangeSelected(range: CalendarRange)`

Would it make sense to at least update the `onDayOfMonthSelected(selectedDay: CalendarDay)` code to utilize this new `updateSelection` function too?

Reply:
Gotcha!